pkg/aspect/accessLoggerManager_test.go

```diff
@@ -19,14 +19,16 @@ import (
 	"testing"
 	"text/template"
 
-	"github.com/golang/protobuf/ptypes/struct"
 	"istio.io/mixer/pkg/adapter"
-	"istio.io/mixer/pkg/aspect/config"
+	aconfig "istio.io/mixer/pkg/aspect/config"
```

Comment:
looks like we could clean this up a bit.  awkward newline between groups of external imports... if we want to keep that, i'd suggest moving the named imports to their own block and otherwise combining.